launcher/image/sync_experiments.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+set -euo pipefail
```

Comment:
We should consider failing silently given that the experiments server has a lower slo (99%) than us.
We shouldn't block the launcher if experiment sync fails. The experiment class is resilient to the lack of a file + lack of expected flags.

Reply:
This is generally good practice. You otherwise don't have error handling here  for exceptional cases.

If you want to avoid failure, you can wrap it in an if statement and log the error or `|| some_failure_logging_func`. If you think bash is not the best language, consider writing your script in Python or another better scripting language.

Reply:
This script has been removed entirely.